install-project.sh

```diff
@@ -1,16 +1,23 @@
-
 slug=$1
-MVNOPTIONS=$2
+MVNOPTIONS="$2 -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/input/dependencies"
```

Comment:
Instead of changing the MVNOPTIONS here, you should directly change the MVNOPTIONS in run_surefire_azure.sh. The reason for this is because scripts like run_surefire_azure.sh makes many more calls to mvn later and they would all need -Dmaven.repo.local set as well.

On second thought though, can you check if there is a global argument that can be set once so that all subsequent calls to mvn would just use the correct -Dmaven.repo.local without needing to provide the argument? If you can globally set it once for mvn then you wouldn't need to change every script's MVNOPTIONS and you can just globally set it once in the run_azure.py script